### DISCUSSION TITLE
Different margins for each item

### DISCUSSION THREAD

**Discussion by rezaaa** [react-grid-layout/react-grid-layout#1869] [Q&A]
Hi

How can I add different margins for each item? I can add a fixed margin on the container but I need to change the default margin for some Items.

<img width="448" alt="image" src="https://user-images.githubusercontent.com/2494766/229072632-0ce6351a-de5c-44c5-90d5-15836b66509b.png">

**Comment by nafishey** [comment]
Have you found a solution yet?
